serviceconnector/src/createServiceConnector/ICreateLinkerContext.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { AuthInfoBase, KnownClientType, LinkerResource } from "@azure/arm-servicelinker";
+import { IStorageAccountWizardContext } from "@microsoft/vscode-azext-azureutils";
+import { ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { TargetServiceType } from "../../constants";
+
+export interface ICreateLinkerContext extends ISubscriptionActionContext, IStorageAccountWizardContext {
+    //Source resource
+    sourceResourceUri?: string;
+    scope?: string; // This is only assigned when the source resource is a container app to indicate which container is being connected
+    runtime?: KnownClientType[];
+    clientType?: KnownClientType;
```

Comment:
Are these properties specific to app service? If so, add a comment to add some context

Reply:
`runtime` is so I will add a comment about that being specific to app service but `clientType` will always be set. In the case of container apps it is set to None but is still being set.